Epsilon.Host.Frontend/src/components/CompetenceProfileLegend.vue

```diff
@@ -0,0 +1,42 @@
+<template>
+    <table class="competence-profile-legend">
+        <tr
+            v-for="level of store.state.domain.valuesSet?.types?.sort((a,b) => a.shortName - b.shortName) as LearningDomainType[]"
+            :key="level"
+            class="competence-profile-legend-row">
+            <td
+                class="kpi-color"
+                :style="{ backgroundColor: '#' + level.hexColor }" />
+            <th class="kpi-text">{{ level.name }}</th>
+        </tr>
+    </table>
+</template>
+
+<script lang="ts" setup>
+import store from "@/store"
+import { LearningDomainType } from "@/api"
+</script>
+
+<style>
+.competence-profile-legend {
+    display: flex;
+    flex-direction: column;
+    width: fit-content;
+    height: fit-content;
+    padding: 2rem;
+}
+
+.kpi-text {
+    font-weight: 400;
+}
+
+.kpi-color {
+    width: 3rem;
+}
```

Comment:
We should probably call it by the name of the type, aka `value` instead of KPI. Also this is not the KPI, but the level of the KPI, but that is only valid in the Fontys context and thus may not be valid when looking at other organizations.